osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,34 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows for using any existing networking that relies on an existing VPC.
```

Comment:
Personal preference: 

s/ allows for using / allows for use of. 

^ That's a personal preference. In most cases I would probably opt for `using` over `use of`, but there are a lot of 'ings' in this sentence that make it read odd to me. 

Also wondering if it should be 'networking' or 'network'. I'm sure you're correct and I'm just being picky :)